### ISSUE TITLE
Do we have a page about the basics of prefixes and CURIEs

### ISSUE THREAD

**Issue by jamesaoverton** [OBOAcademy/obook#355]
I couldn't immediately find one, and I don't have time to make one right now, but here's a reply I just made on the Ontobee Discuss mailing list:

Yes, it's common practise in the OBO community to define a prefix "OHD" which expands to "http://purl.obolibrary.org/obo/OHD_", and then use identifiers (called "CURIE"s) such as "OHD:0000005".

There's a good technical reason why the "#" is appropriate for small ontologies by not for large ones. When you put the URL "http://www.w3.org/2000/01/rdf-schema#label" into your browser, it will fetch the full resource at "http://www.w3.org/2000/01/rdf-schema" and then scroll to a section identified by the part after "#", e.g. "label". RDF Schema is small, so downloading all of "http://www.w3.org/2000/01/rdf-schema" every time is not a problem. But many OBO ontologies are much larger, dozens or hundreds of megabytes, so this approach becomes infeasible.

Instead we provide a URL to download the full resource, e.g. the OWL file at "http://purl.obolibrary.org/obo/ohd.owl", and also URLs for each term, such as "http://purl.obolibrary.org/obo/OHD_0000005". The term URL identifies the term and also links to an ontology term browser (such as Ontobee) that provides more information about that term.

**Comment by matentzn** [comment]
Very interesting lesson, we should really at least draft a page real quick to deal with the basics here.

**Comment by matentzn** [comment]
Fold in https://cthoyt.com/2021/09/14/curies.html

**Comment by balhoff** [comment]
> Fold in https://cthoyt.com/2021/09/14/curies.html

I don't really agree with the perspective there. It's true that not all kinds of IDs have been provided with a canonical URI namespace, but when they have, it works very well. The examples given actually don't use the CURIE prefixes most commonly found (`go:0032571` instead of `GO:0032571`) anyway.

**Comment by jamesaoverton** [comment]
@balhoff Agreed: in OBO docs (and wherever people will listen) we need to push the OBO standard prefixes, which have been in place for decades.

**Comment by matentzn** [comment]
The tutorial will reflect the OBO perspective!
